radiacode.rules

```diff
@@ -0,0 +1,2 @@
+# Radiacode-10X radiation detector
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="f123", ATTRS{manufacturer}=="AngioScan", GROUP="plugdev", MODE="0660"
```

Comment:
`plugdev` group is ubuntu specific.

[arch wiki](https://wiki.archlinux.org/title/udev#Allowing_regular_users_to_use_devices) recommends to use `TAG+="uaccess"` instead, which should work on any systemd-based distro for any logged-in user.

Reply:
Done